server/store/datastore/server_config.go

```diff
@@ -43,7 +43,9 @@ func (s storage) ServerConfigSet(key, value string) error {
 		return err
 	}
 
-	_, err = s.engine.Where("key = ?", config.Key).AllCols().Update(config)
+	_, err = s.engine.Cols("value").Update(config, &model.ServerConfig{
```

Comment:
we should not change the coding style here ...

it'S first explite filter ... then opt cols to update ... then the changed object

Reply:
Yes in this case it **IS** an xorm bug

We can merge your sol. but witj a todo connent to an open upstream bug report